pytorch_lightning/distributed/dist.py

```diff
@@ -18,6 +18,12 @@
 
 
 class LightningDistributed:
+    """
+    .. deprecated:: v1.5
+        This method is deprecated in v1.5and will be removed in v1.7.
+        Logic moved to ddp and ddpspawning
```

Comment:
```suggestion
        This class is deprecated in v1.5 and will be removed in v1.7.
        The broadcast logic will be moved to the :class:`DDPPlugin` and  :class`DDPSpawnPlugin` classes.
```

Reply:
@four4fish we need to print a rank_zero_deprection message here in the init of the class and/or in the broadcast message below